services/QuillLMS/db/migrate/20221110072939_create_evidence_prompt_healths.evidence.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# This migration comes from evidence (originally 20221110063922)
+
+
+class CreateEvidencePromptHealths < ActiveRecord::Migration[6.1]
+  def change
+    create_table :evidence_prompt_healths do |t|
+      t.integer     :prompt_id, null: false
+      t.string      :activity_short_name, null: false
+      t.string      :text, null: false
+      t.integer     :current_version, null: false
+      t.integer     :version_responses, null: false
+      t.integer     :first_attempt_optimal
+      t.integer     :final_attempt_optimal
+      t.float       :avg_attempts
+      t.float       :confidence
+      t.integer     :percent_automl_consecutive_repeated
+      t.integer     :percent_automl
+      t.integer     :percent_plagiarism
+      t.integer     :percent_opinion
+      t.integer     :percent_grammar
+      t.integer     :percent_spelling
+      t.integer     :avg_time_spent_per_prompt
+
+      t.references  :evidence_activity_health
+
+      t.timestamps
+    end
+    add_foreign_key :evidence_prompt_healths, :evidence_activity_healths, column: :evidence_activity_health_id, on_delete: :cascade
```

Comment:
Interesting, I've always just used `dependent: :destroy` on the model.  What is the reasoning for also including the `on_delete: :cascade`?

Reply:
Hmm I've always just done it on both the migration and the model, I'm not sure if there's a plus to doing it on both honestly. From [this answer here](https://stackoverflow.com/questions/12556614/rails-delete-cascade-vs-dependent-destroy) it seems like there might be performance benefits to `on_delete: cascade`?